soc/arm/microchip_mec/Kconfig

```diff
@@ -11,6 +12,212 @@ config SOC_FAMILY
 	string
 	default "microchip_mec"
 
+menuconfig MCHP_MEC_UNSIGNED_HEADER
+	bool "Create an unsigned output binary with MCHP MEC binary header"
+	help
+	  On Microchip MEC series chip, the ROM code loads firmware image from flash
+	  to RAM using a TAG to locate a Header which specifies the location and
+	  size of the firmware image. Enable this to invoke the mec_spi_gen tool
+	  which generates an SPI image with TAG, Header, and firmware binary. This
+	  tool does not produce a signed image which can be authenticated by the
+	  Boot-ROM. Use the full Microchip SPI image generator program for
+	  authentication and all other Boot-ROM loader features. Refer to the MCHP
+	  EVB boards for an example.
+
+if MCHP_MEC_UNSIGNED_HEADER
+
+config MCHP_MEC_IMAGE_OUTPUT_BIN
```

Comment:
Do you need this option? This whole block depends on `MCHP_MEC_UNSIGNED_HEADER`, but then we have this as the condition for calling `mec_spi_gen.py`. That seems redundant and unnecessary since you could go with `MCHP_MEC_IMAGE_OUTPUT_BIN=n` then the options below here are meaningless.

How about keeping only the menuconfig option and use that as condition for calling the builder script?

Reply:
removed `config MCHP_MEC_IMAGE_OUTPUT_BIN`